typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -1381,13 +1399,15 @@ class WellLogView
 
         if (
             this.props.welllog !== prevProps.welllog ||
+            this.props.wellLogSets !== prevProps.wellLogSets ||
```

Comment:
Same as above comment.
if (this.props.wellLogSets ?? this.props.welllog !== prevProps.wellLogSets ?? prevProps.welllog) return true;